src/strings/PowerFxResources.en-US.resx

```diff
@@ -5158,4 +5158,15 @@
     <value>Can't delegate {0}: contains a behavior function '{1}'.</value>
     <comment>Warning message.</comment>
   </data>
+  <data name="AboutPlainText" xml:space="preserve">
+    <value>The PlainText function removes HTML and XML tags.</value>
+    <comment>Description of 'PlainText' function.</comment>
+  </data>
+  <data name="PlainTextArg1" xml:space="preserve">
+    <value>PlainText</value>
```

Comment:
>PlainText [](http://example.com/codeflow?start=11&length=9)

`text` (this is the text that is going to be "plainified")

Reply:
update